server/src/main/java/org/opensearch/gateway/AsyncShardFetchCache.java

```diff
@@ -164,6 +167,7 @@ Map<DiscoveryNode, K> getCacheData(DiscoveryNodes nodes, Set<String> failedNodes
     void processResponses(List<K> responses, long fetchingRound) {
         for (K response : responses) {
             BaseNodeEntry nodeEntry = getCache().get(response.getNode().getId());
+            clusterManagerMetrics.incrementCounter(clusterManagerMetrics.asyncFetchTotalCounter, 1.0);
```

Comment:
should this be a successful fetch count - ? 
Can we increment the counter by `reponses.size` ?

Reply:
yeah we can change this to successful fetch count instead